packages/help-center/src/hooks/use-should-render-chat-option.tsx

```diff
@@ -3,34 +3,41 @@ import { useHappychatAvailable } from '@automattic/happychat-connection';
 
 type Result = {
 	render: boolean;
-	state?: 'AVAILABLE' | 'UNAVAILABLE' | 'CLOSED';
+	state: 'AVAILABLE' | 'UNAVAILABLE' | 'CLOSED';
 	isLoading: boolean;
+	eligible: boolean;
 };
 
 export function useShouldRenderChatOption(): Result {
 	const { data: chatStatus } = useSupportAvailability( 'CHAT' );
 	const { data, isLoading } = useHappychatAvailable();
+
 	if ( ! chatStatus?.is_user_eligible ) {
 		return {
 			render: false,
 			isLoading,
+			state: chatStatus?.is_chat_closed ? 'CLOSED' : 'UNAVAILABLE',
+			eligible: false,
```

Comment:
`eligible` and `render` are the same thing, I just added `eligible` for more readibility.